k8s/helm/hypercode/templates/orchestrator-deployment.yaml

```
@@ -0,0 +1,54 @@
+    {{- include "hypercode.labels" . | nindent 4 }}
+    app.kubernetes.io/component: orchestrator
+spec:
+  replicas: {{ .Values.replicaCount.orchestrator }}
+  selector:
+    matchLabels:
+      {{- include "hypercode.selectorLabels" . | nindent 6 }}
+      app.kubernetes.io/component: orchestrator
+  template:
+    metadata:
+      labels:
+        {{- include "hypercode.selectorLabels" . | nindent 8 }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)